composer.json

```diff
@@ -108,7 +108,7 @@
     "phpunit": "vendor/bin/phpunit",
     "security-tests": [
       "wget -q https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_amd64 -O local-php-security-checker && chmod +x ./local-php-security-checker && ./local-php-security-checker",
-      "yarn audit --groups dependencies optionalDependencies"
+      "yarn audit --level=8"
```

Comment:
In other projects we do not specify the level, why did you chose to do so here?

Reply:
This was a decision made by my ex colleague Eric. Due to the vast amount of peer dependencies, this level op 8 was chosen.